tasks/update-mlnx-ofed.yml

```
@@ -0,0 +1,19 @@
+---
+ - name: Add script content
+   ansible.builtin.copy:
+     dest: /etc/kernel/header_postinst.d/mlnx-ofed-kernel
```

You may want to prefix this file with `01-` to execute before `/etc/kernel/header_postinst.d/dkms`

```suggestion
     dest: /etc/kernel/header_postinst.d/01-dkms-mlnx-ofed-kernel
```